src/libraries/System.Runtime/tests/System/ByteTests.GenericMath.cs

```diff
@@ -55,6 +55,17 @@ public static void op_AdditionTest()
             Assert.Equal((byte)0x00, AdditionOperatorsHelper<byte, byte, byte>.op_Addition((byte)0xFF, (byte)1));
         }
 
+        [Fact]
+        public static void op_CheckedAdditionTest()
```

Comment:
Imo, the whole thing would be a bit more structured if you made the Equality Checks a Theory with InlineData and the Throws Check a separate Fact.

Reply:
The tests are explicitly simple and easily copyable here since I have to replicate it for each of many types.

These are basically just testing that the interface forwards down correctly and so don't need a lot of complexity.